### DISCUSSION TITLE
Use MaxLength Plugin have Error

### DISCUSSION THREAD

**Discussion by Michael-indochat** [facebook/lexical#4891] [Q&A]
### Settings
MaxLength: 30
### Description:
I encountered a bug while using the MaxLengthPlugin. When I paste a string containing more than 30 characters, it behaves as expected the first time, truncating the text to the specified maximum length. However, when I continue to paste the same long text, an error occurs, rendering the editor unresponsive.

### Demonstration:
https://github.com/facebook/lexical/assets/114893149/079ef7de-0052-44fc-8114-72722d580f2a

### Implementation Details:
I am utilizing the [MaxLengthPlugin](https://github.com/facebook/lexical/blob/cf0b053a9694af3a6beac42f5fd581c01e5c7a6a/packages/lexical-playground/src/plugins/MaxLengthPlugin/index.tsx#L15) with the following code snippet:

```typescript
<LexicalComposer initialConfig={initialConfig}>
  <div className="editor-container">
    <PlainTextPlugin
      contentEditable={<ContentEditable className="editor-input" />}
      placeholder={<Placeholder />}
      ErrorBoundary={LexicalErrorBoundary}
    />
    <MaxLengthPlugin maxLength={30} />
<LexicalComposer/>
```

### Request:
I am looking for assistance in resolving this issue, as it hampers the functionality of the editor. Any insights or guidance on how to fix this would be greatly appreciated.

**Comment by rik-iso** [comment]
+1 - and setting it on the contenteditable doesnt help
